docs/guide/distancesensor-sensors.md

```diff
@@ -0,0 +1,146 @@
+## DistanceSensor Sensors
+
+Some predefined [DistanceSensor](../reference/distancesensor.md) models are present in Webots such as some infrared sensors.
+
+Nevertheless, adding a generic [DistanceSensor](../reference/distancesensor.md) node is sufficient for most of the applications because they are customizable (resolution, lookupTable, aperture, etc.).
```

Comment:
For me these two sentences are not very clear (this one and the previous one).
What is the difference between a predefined `DistanceSensor` and a generic `DistanceSensor`?

Personally I think that if you want to also mention the generic `DistanceSensor` node, then you have to do it first at the very beginning of the section.
Then when writing about predefined sensors (like the Sharp one), you should not link to the `DistanceSensor` node reference page.